config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,18 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// BareMetalPlatformStatus holds the current status of the BareMetal infrastructure provider.
+type BareMetalPlatformStatus struct {
+	// api_vip holds the Virtual IP (VIP) that can be used to contact the API
+	APIVIP string `json:"api_vip"`
+
+	// ingress_vip holds the VIP for Ingress
```

Comment:
separate fields don't display together.  You'll need to spell our virtual IP.  This isn't clear to me about what IPs this maps to.